get_genotype_vcf.py

```diff
@@ -127,12 +159,15 @@ def main(vds_version, chromosomes, cv_maf_threshold, vre_mac_threshold, vre_n_ma
             bcftools_job.image(BCFTOOLS_IMAGE)
             bcftools_job.cpu(4)
             bcftools_job.storage('15G')
-            # remove chr
+            # now remove "chr" from chromosome names
+            rename_file = create_rename_chr(rename_file)
             bcftools_job.command(
-                f'bcftools annotate --remove-chrs {vcf_input} | bgzip > {vcf_input}'
+                f'bcftools annotate --rename-chrs {rename_file} {vcf_input} -o {bcftools_job.vcf}'
```

Comment:
```suggestion
                f'for num in  {1..22} X Y; do echo "chr${num} ${num}" >> chr_update.txt; done && bcftools annotate --rename-chrs chr_update.txt {vcf_input} -o {bcftools_job.vcf}'
```

Reply:
Not sure this is the best one-liner, but the command 
```for num in  {1..22} X Y; do echo "chr${num} ${num}" >> chr_update.txt; done```
will generate the file you want as a local file